Scripts/ChangeRemediationSLAOnSevChange.yml

```diff
@@ -0,0 +1,43 @@
+commonfields:
+  id: 310ed078-8188-443f-83cc-7cb80c55971d
```

Comment:
update the id, and update the script name to `script-ChangeRemediationSLAOnSevChange.yml`